addons/survey/controllers/main.py

```diff
@@ -728,39 +723,137 @@ def _get_user_input_domain(self, survey, line_filter_domain, **post):
             user_input_domain = expression.AND([[('scoring_success', '=', False)], user_input_domain])
         elif post.get('passed'):
             user_input_domain = expression.AND([[('scoring_success', '=', True)], user_input_domain])
-
         return user_input_domain
 
     def _extract_filters_data(self, survey, post):
+        """
+            Extracts the filters from the URL to returns the related user_input_lines and
+            the parameters used to render/remove the filters on the results page (search_filters).
+
+            :rtype: (recordset, List[dict])
```

Comment:
Also here (see existing methods in this exact file), I wouldn't have this blank line, nor the indentation.

```suggestion
        """Extracts the filters from the URL to returns the related user_input_lines and
        the parameters used to render/remove the filters on the results page (search_filters).

        :rtype: (recordset, List[dict])
```